browser/components/CodeEditor.js

```diff
@@ -28,6 +28,7 @@ import {generateInEditor, tocExistsInEditor} from 'browser/lib/markdown-toc-gene
 import markdownlint from 'markdownlint'
 import Jsonlint from 'jsonlint-mod'
 import { DEFAULT_CONFIG } from '../main/lib/ConfigManager'
+const prettier = require('prettier')
```

Comment:
Can you replace this with `import` statement?

Reply:
Addressed in #bde357f

Reply:
Great!